GitCommands/Git/GitModule.cs

```diff
@@ -1812,6 +1828,60 @@ public string UnstageFiles(IReadOnlyList<GitItemStatus> files)
             return output.ToString();
         }
 
+        /// <summary>
+        /// Batch unstage files without outputing shouldRescanChanges variable
+        /// </summary>
+        /// <param name="selectedItems">Selected file items</param>
+        /// <param name="action">Progress update callback</param>
+        public void BatchUnstageFiles(IEnumerable<GitItemStatus> selectedItems, Action<BatchProgressEventArgs> action = null)
+        {
+            BatchUnstageFiles(selectedItems, out var _, action);
+        }
+
+        /// <summary>
+        /// Batch unstage files using <see cref="ExecutableExtensions.RunBatchCommand(IExecutable, ICollection{BatchArgumentItem}, Action{BatchProgressEventArgs}, byte[], bool)"/>
+        /// </summary>
+        /// <param name="selectedItems">Selected file items</param>
+        /// <param name="shouldRescanChanges">Rescan change flag</param>
+        /// <param name="action">Progress update callback</param>
+        public void BatchUnstageFiles(IEnumerable<GitItemStatus> selectedItems, out bool shouldRescanChanges, Action<BatchProgressEventArgs> action = null)
```

Comment:
This API doesn't look good. I don't see why we need an `out` variable:
```suggestion
        /// <returns><see langword="true" /> if changes should be rescanned; otherwise <see langword="false" />.
        public bool BatchUnstageFiles(IEnumerable<GitItemStatus> selectedItems, Action<BatchProgressEventArgs> action = null)
```

And the overload is now redundant.